.vib/jaeger/goss/goss.yaml

```diff
@@ -0,0 +1,20 @@
+http:
+  http://jaeger--agent:{{ .Vars.agent.service.ports.admin }}/:
+    status: 200
+  http://jaeger-collector:{{ .Vars.collector.service.ports.admin }}/:
+    status: 200
+  http://jaeger-query:{{ .Vars.query.service.ports.admin }}/:
+    status: 200
+file:
+  /opt/bitnami/jaeger/bin/jaeger-agent:
+    exists: true
+    filetype: file
+    mode: "0755"
+    owner: root
```

Comment:
We used this approach in our first tests, but then we realised that:

- If the binary does not exist, the chart would fail to deploy
- Checking the ownership and permissions does make sense on a docker-image level, because this is not something we actively change in the chart.

So we no longer write these types of tests because they are not really valuable 😁

Reply:
Removed [32ab3ee](https://github.com/bitnami/charts/pull/13480/commits/32ab3ee6bfb9ed79a64029e3a49e59ffe1aa00f1)